examples/comments.pi

```diff
@@ -0,0 +1,4 @@
+-- This is a comment 
+record {
+  -- Another comment 
+} : Record {}
```

Comment:
Oooh thanks for the test! Really need to get a good integration test harness set up at some stage 🤔